src/mime.cc

```diff
@@ -234,17 +234,14 @@ mimeInit(char *filename)
     char buf[BUFSIZ];
```

Comment:
While we're at it, would it make sense to make this static?

Reply:
If we were doing a larger refactor of the mime code that would be one of the actions. But changing the I.O buffer is out of scope for just using RegexPattern.